WalletWasabi/Helpers/PasswordHelper.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using WalletWasabi.KeyManagement;
+using NBitcoin;
+using System.Security;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Helpers
+{
+	public static class PasswordHelper
+	{
+		public const int MaxPasswordLength = 150;
+		public const string CompatibilityPasswordWarnMessage = "Compatibility password was used! Please consider generating a new wallet to ensure recoverability!";
+		public static string PasswordTooLongMessage = $"Password too long (Max {MaxPasswordLength} characters).";
```

Comment:
Why is this variable `static` and not `const`?

Reply:
Combining int and string implicitly calls ToString(). So it cannot be combined at compile time, cannot be const.

Reply:
But I put there a readonly.